### ISSUE TITLE
warning

### ISSUE THREAD

**Issue by stevemmarshall** [bardin-lab/readtagger#281]
Hi

Have you seen this issue with pysam?

/opt/conda/lib/python3.7/importlib/_bootstrap.py:219: RuntimeWarning: pysam.libcalignedsegment.PileupColumn size changed, may indicate binary incompatibility. Expected 72 from C header, got 88 from PyObject

Cheers
Steve

**Comment by mvdbeek** [comment]
Yeah, I've seen this for quite some time with pysam installed via Conda. It doesn't seem to affect results. While it looks worrying I'm ignoring this for now.
